### PR TITLE
chore: Misc minor things

### PR DESCRIPTION
**Pull request by rschristian**
Sorry, hodge-podge of stuff I came across again when working on Vite. Going by the commits is probably easiest.

### CODE REVIEW

File: content/de/about/project-goals.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Preact zielt darauf ab, Leistung aufgrund folgender Ziele zu liefern:
 
 Einige Features von React wurden absichtlich nicht in Preact integriert, da sie entweder mit Berücksichtigung der oben genannten primären Projektziele nicht umsetzbar sind oder aber nicht in den Umfang von Preacts Grundausstattung an Funktionalitäten passen.
 
-- Die beabsichtigten Artikel unter [Was fehlt?](/guide/differences-to-react#whats-missing):
+- Die beabsichtigten Artikel unter [Was fehlt?](/guide/v8/differences-to-react#whats-missing):
```
Comment (rschristian) [inline content/de/about/project-goals.md:22]: Unversioned links are a problem for prerendering w/ link discovery, as they just 404.

File: content/en/guide/v10/server-side-rendering.md [hunk 2]
```diff
@@ -7,8 +7,6 @@ description: 'Render your Preact application on the server to show content to us
 
 Server-Side Rendering (often abbreviated as "SSR") allows you to render your application to an HTML string that can be sent to the client to improve load time. Outside of that there are other scenarios, like testing, where SSR proves really useful.
 
-> Note: SSR is automatically enabled with `preact-cli` :tada:
-
```
Comment (rschristian) [inline content/en/guide/v10/server-side-rendering.md:11]: Eventually I'd like to write up a page detailing SSR w/ the Vite preset (especially as I crammed _a lot_ of info into the preset's readme) but... not today.

File: src/lib/github.js [hunk 3]
```diff
@@ -10,8 +10,8 @@ export const repoInfo = repo =>
 		.then(checkStatus)
 		.then(r => r.json())
 		.catch(() => ({
-			stargazers_count: 9999,
-			watchers_count: 9999
+			stargazers_count: 35780,
+			watchers_count: 405
```
Comment (rschristian) [inline src/lib/github.js:14]: I mentioned this in Slack, but our lambda functions are current borked in prod, resulting in this fallback data always shown, which is quite incorrect.

File: content/ru/index.md [hunk 4]
```diff
@@ -217,7 +217,7 @@ document.body
 </code></pre>
 
 <div class="home-demo">
-<github-stars simple user="preactjs" repo="preact"></github-stars>
+<github-stars user="preactjs" repo="preact"></github-stars>
```
Comment (rschristian) [inline content/ru/index.md:220]: Missed in #1067, however, it looks _a lot_ better IMO? I don't see a need for `simple`.

| Before / 'simple' Stars | After|
| --- | --- |
| Simple Stars | temp2 |

Comment (rschristian) [reply content/ru/index.md:220]: TODO: Example code needs to be updated too

File: src/components/header/style.module.css [hunk 5]
```diff
@@ -483,7 +483,7 @@
 
 :global(.aa-dropdown-menu) {
 	position: fixed !important;
-	top: 60px !important;
+	top: var(--header-height) !important;
```
Comment (rschristian) [inline src/components/header/style.module.css:486]: `--header-height` is `3.5.rem`/`56px` by default, which 60px is pretty close to, but any level of zoom and it'll start overlaying incorrectly.

**Review by marvinhagemeister — APPROVED** [review]

**Pull request merged**